apps/apple/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 platform :ios, '12.0'
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
```

Comment:
There were two possible solutions here, but this one avoids duplicating node_modules if we can just redirect it to root's version.